models/public/forward_tacotron/description/forward_tacotron.md

```diff
@@ -0,0 +1,123 @@
+# ForwardTacotron
+
+## Use Case and High-Level Description
+
+ForwardTacotron is model for text to speech task originally trained on PyTorch\*
+then converted to ONNX\* format. Forward Tacotron is the model for mel-spectrogramm regression from text.
+For details see [paper](https://arxiv.org/pdf/1703.10135.pdf),[paper](https://arxiv.org/pdf/1905.09263.pdf), [repository](https://github.com/as-ideas/ForwardTacotron).
+
+### Steps to Reproduce PyTorch to ONNX Conversion
```

Comment:
I think it's misleading to just call this "conversion" when it also includes training. You should change it to "training & conversion" and also clarify in the description that this model has been retrained.